src/datasets/utils/resources/creators.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
these files were brought from https://github.com/huggingface/datasets-tagging

Reply:
Is it possible to have these json files in a `resources` folder at the root of the repo or something like that ?
So that it's easier to find them

Is that compatible with how importlib.resources works ? Or do they have to be inside the `src/datasets` directory ?

Reply:
I find that it being in the same directory as the schema definition is better, easier to find for lib devs and for the others the url to the directory would be enough. 
wdyt?